core/pom.xml

```diff
@@ -215,6 +215,11 @@
       <version>3.2.10</version>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-scala_2.10</artifactId>
+      <version>2.3.1</version>
```

Comment:
not exactly what you're asking but this module doesn't always play nicely with different versions of jackson itself.  So if somehow jackson gets bumped up to a later version b/c of some transitive dependency, often this needs to get bumped up to go along with it.